pkg/controller/controller_binding.go

```diff
@@ -175,6 +175,11 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 		return fmt.Errorf("Ongoing Asynchronous operation")
 	}
 
+	if instance.Spec.ServiceClassRef == nil || instance.Spec.ServicePlanRef == nil {
+		// retry later
+		return fmt.Errorf("ServiceClass or ServicePlan references for Instance have not been resolved yet")
```

Comment:
This seems like a situation we'd want to update the instance credential's status. If the instance's ServiceClassRef and ServicePlanRef are nil, that would mean the instance couldn't have been provisioned yet, which is useful information for the user.

Or are there any other situations where they would be nil?

Reply:
Well, this case is analogous to what was there before in that a ServiceClass / Plan was not there and the comment for retry later was exactly the same, so semantically I think it's what we wanted before so I left it the same as it was.

Reply:
I think this is the same condition as "service instance isn't ready"